frontend/src/widgets/TaskCard/taskcard.jsx

```diff
@@ -0,0 +1,30 @@
+/** @jsxImportSource @emotion/react */
+
+import Card from 'components/Card';
+
+// Style
+import * as style from './taskcard-styles';
+
+export default function TaskCard({ children, title, body, assignee, tag, ...passedProps }) {
+  return (
+    <div css={style.taskCard} {...passedProps}>
```

Comment:
instead of wrapping the Card in a div, much better if Card nalang dayon and didto i pasa ang css ug ang passedProps